docs/yaml_docs/pipeline_visualization_yml.md

```diff
@@ -0,0 +1,161 @@
+  - <span class="parameter">rna:</span>`String`, Default: logged_counts<br>
+  - <span class="parameter">prot:</span>`String`, Default: clr<br> 
+  - <span class="parameter">atac:</span>`String`, Default: signac_norm<br> 
+Check [gene_list_format.md](https://github.com/DendrouLab/panpipes/edit/clustering_g/docs/usage/gene_list_format.md) for Plot marker csv format instructions. 
+
+
+## Plot metadata variables 
+
+- <span class="parameter">categorical_vars:</span>`String`, Default: &categorical_vars<br>
+  - <span class="parameter">all:</span>`String`, Default: rep:receptor_subtype  sample_id<br>
+Metrics to be plotted on every modality. 
+  - <span class="parameter">rna:</span>`String`, Default: rna:predicted_doublets  rna:phase<br>
+  - <span class="parameter">prot:</span>`String`, Default: prot:leiden_res0.2    prot:leiden_res1<br>
+  - <span class="parameter">atac:</span>`String`, Default: <br>
+  - <span class="parameter">rep:</span>`String`, Default: rep:has_ir<br>
+  - <span class="parameter">multimodal:</span>`String`, Default: leiden_totalVI    mdata_colsr<br> 
+
+- <span class="parameter">continuous_vars:</span>`String`, Default: &continuous_vars<br>
+  - <span class="parameter">all:</span>`String`, Default:leiden_res0.5<br>
+Metrics to be plotted on every modality. 
+  - <span class="parameter">rna:</span>`String`, Default: rna:total_counts<br>
+  - <span class="parameter">prot:</span>`String`, Default: prot:total_counts<br>
+  - <span class="parameter">atac:</span>`String`, Default: <br>
+  - <span class="parameter">multimodal:</span>`String`, Default: rna:total_counts    prot:total_counts<br>
+  
+- <span class="parameter"paired_scatter:</span>`String`, Default: scatter_features.csv<br>
+Check [gene_list_format.md](https://github.com/DendrouLab/panpipes/edit/clustering_g/docs/usage/gene_list_format.md) for metadata csv format instructions. 
+
+## Plot style 
+Choose the plot type desired. 
+- <span class="parameter">do_plots:</span><br>
+  - <span class="parameter">categorical_barplots:</span>`Boolean`, Default: True<br>
+  Plot each categorical variable as a bar plot. 
+  - <span class="parameter">categorical_stacked_barplots:</span>`Boolean`, Default: True<br>
+  Plot each grouping var as a bar plot, with categorical variables stacked.
+  - <span class="parameter">continuous_violin:</span>`Boolean`, Default: True<br>
+  Plot each continuous variable as a violin plot.
+  - <span class="parameter">marker_dotplots:</span>`Boolean`, Default: True<br>
+  Plots a marker dotplots as produced by scanpy.pl.dotplot.
+  - <span class="parameter"> marker_matrixplots:</span>`Boolean`, Default: True<br>
+  Plots marker matrixplot as produced by scanpy.pl.matrixplot.
+  - <span class="parameter">paired_scatters:</span>`Boolean`, Default: True<br>
+  Plots scatter plots as defined in paired_scatters csv file (scatter_features.csv).
+
+- <span class="parameter">embedding:</span><br>
+Define all embedding for the plots and modalities.  
```

Comment:
can you put exactly what is in the pipeline.yml here:  embedding plots (e.g. UMAP, PCA)  using the modality and embedding basis specified. This  will plot all of minimal markers csv, and categorical, and continuous variables